src/layout/Titlebar/Titlebar.container.js

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect } from 'react';
-import { Titlebar } from './Titlebar';
+import Titlebar from './Titlebar';
 
 const EnhancedTitlebar = props => {
   useEffect(() => {}, []);
```

Comment:
not used ?

Reply:
No, but I prefer keep the container file with a base structure for this type of file.